[Integration]: Quickbooks Integration

Smyja opened the pull request:
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #5336 

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?
https://www.loom.com/share/887094aa585a4f0caae6eca6a86f6541

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

ZoranPandovski reviewed (changes requested):
Can you please share screenshots or a short video about how this works? Also, I am confused how it works without the native query implementation?

Smyja commented:
> Can you please share screenshots or a short video about how this works? Also, I am confused how it works without the native query implementation?

Here's a loom link, i have updated the description also and added the native query.
https://www.loom.com/share/887094aa585a4f0caae6eca6a86f6541

ZoranPandovski commented on mindsdb/integrations/handlers/quickbooks_handler/quickbooks_handler.py:
```diff
@@ -0,0 +1,98 @@
+            quickbooks = self.connect()
+            quickbooks.accounts.get()
+            log.logger.info(quickbooks.accounts.get())
+            print('Connected to Quickbooks API')
```
Please remove the print

Smyja replied:
done

ZoranPandovski commented on mindsdb/integrations/handlers/quickbooks_handler/quickbooks_handler.py:
```diff
@@ -0,0 +1,98 @@
+            log.logger.error(response.error_message)
+
+        if response.success is False and self.is_connected is True:
+            print('Disconnected from Quickbooks API')
```
Same here

Smyja replied:
done

Smyja commented:
I have resolved the issues @ZoranPandovski
